app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,241 @@
+class RelationshipsController < ApplicationController
+  delegate :relationship_form_repository, :all_relationships_query, :to => :injector
+  delegate :deprecate_relationship_form_repository, :to => :deprecate_injector
+  delegate :term_form_repository, :to => :term_injector
+  skip_before_filter :require_admin, :only => [:review_update, :mark_reviewed]
+
+  include GitInterface
+  def index
+    #Grab all relationships
+    @relationships = all_relationships_query.call
+  end
+
+  def new_parent_relationship
```

Comment:
Just leaving a note related to the discussion of condensing these actions into 1, using a select (or hidden input) to set `hier_parent` or `heir_child` when submitting the form. Eliminate duplicate related views, routes, etc.